nestkernel/connection_manager.cpp

```diff
@@ -1117,14 +1117,7 @@ nest::ConnectionManager::get_connections(
 
   if ( source == 0 and target == 0 )
   {
-#ifdef _OPENMP
-#pragma omp parallel
-    {
-      thread t = kernel().vp_manager.get_thread_id();
-#else
-    for ( thread t = 0; t < kernel().vp_manager.get_num_threads(); ++t )
-    {
-#endif
```

Comment:
this for loop is replaced with `size_t t = 0;` in your macro. doesn't this break the logic? or was the for loop unnecessary anyway?